src/main/java/reposense/dataobject/GithubIssue.java

```diff
@@ -0,0 +1,36 @@
+package reposense.dataobject;
+
+import java.util.List;
+
+/**
+ * Represents a github issue
+ */
+public class GithubIssue {
+
+    private final GithubIssueTitle title;
+    private final List<GithubIssueLabels> labels;
+    private final GithubIssueIndex index;
+    private final GithubIssueState state;
```

Comment:
If all these classes are all just going to be `String`, I not sure is it worth it to create a model for each of them.
This class on its own is also similar to `GHIssue` from `org.kohsuke.github`. Why not just use `GHIssue` instead?